spell mistake change - change detection (#47261)
angular/angular: aio/content/guide/change-detection.md

**Change detection** is the process through which Angular checks to see whether your application state has changed, and if any DOM needs to be updated. At a high level, Angular walks your components from top to bottom, looking for changes. Angular runs its change detection mechanism periodically so that changes to the data model are reflected in an app’s view. Change detection can be triggered either manually or through an asynchronous event (for example, a user interaction or an XHR completion).

> Change detection is a highly optimized performance, but it can still cause slowdowns if the application runs it too frequently.

Reviewer: ```suggestion
Change detection is optimized for performance, but it can still cause slowdowns if the application runs it too frequently.
```
